base/src/org/adempiere/process/InvoiceGenerateFromShipment.java

```diff
@@ -48,9 +49,33 @@
  *  @author Yamel Senih, ysenih@erpcya.com, ERPCyA http://www.erpcya.com
  *		<a href="https://github.com/adempiere/adempiere/issues/514">
  * 		@see FR [ 514 ] Error in process to Generate invoices from shipments</a>
+ *  @author https://github.com/homebeaver
+ *	@see <a href="https://github.com/adempiere/adempiere/issues/1657"> 
+ * 		  bug in swing client "Generate Invoices from Shipments (manual)"</a>
  */
-public class InvoiceGenerateFromShipment extends InvoiceGenerateFromShipmentAbstract {
+public class InvoiceGenerateFromShipment extends SvrProcess {
 	
+	/**	Manual Selection		*/

```

Comment:
Hi @homebeaver it is unnecessary with new approach of Abstract class for process